lib/group_consumer.js

```diff
@@ -196,7 +201,7 @@ GroupConsumer.prototype._fullRejoin = function () {
             });
         })
         .catch(function (err) {
-            self.client.error('Full rejoin attempt failed:', err);
+            self.client.error('Full rejoin attempt failed:', JSON.stringify(err));
```

Comment:
@mnorkin why do you stringily error here?

Reply:
Removed